src/app/api/users/route.ts

```diff
@@ -0,0 +1,55 @@
+import prisma from "@/prisma/client";
```

Comment:
@frattezi  @Luana-ns  I'm having issues to find prisma/client 😢

Reply:
let's avoid importing `prisma` directly. instead of that, use the shared `db` instance.